pkg/logs/interface.go

```diff
@@ -3,7 +3,16 @@ package logs
 import "io"
 
 type Client interface {
-	// DevModeLogs gets logs for the provided component name and namespace. A component could have multiple pods and
-	// containers running on the cluster. It returns a slice of maps where container name is the key and its logs are the value
+	// DevModeLogs gets logs for the Dev mode containers of the provided component name and namespace. A component could
```

Comment:
I find this interface (and the underlying code) very repetitive. I would prefer to see a single method, with a Mode parameter

Reply:
Done.